watch.ts

```diff
@@ -0,0 +1,99 @@
+import ts = require("typescript");
+import {compileProgram, CompilerOptions} from "./compile";
+import {basename, dirname} from "path";
+import fs from "fs";
+import {assemble} from "./assembler";
+import {ObjectToDesyncedString} from "./dsconvert";
+
+Promise.all([
+    import("clipboardy")
```

Comment:
I got the error:

```
TS1479: The current file is a CommonJS module whose imports will produce require calls; however, the referenced file is an ECMAScript module and cannot be imported with require. Consider writing a dynamic import(clipboardy)' call instead.
To convert this file to an ECMAScript module, change its file extension to .mts, or add the field `type: module` to M:/tools/desynced-tools/package.json
```

Not sure what the best approach is:

* Dynamic import
* Rename to `mts`
* Add `type: module` to `package.json`

Reply:
Ah, I think this is because of the module setting in tsconfig. I think I had that for esbuild, but it causes problems in node. Maybe we need to have a separate tsconfig for the browser build? Anyway, this seems fine for now.